src/TestInfrastructure/Helpers/RuleSetAssert.cs

```diff
@@ -23,8 +23,8 @@ public static void AreEqual(RuleSet expected, RuleSet actual, string message = n
 
             string nl = Environment.NewLine;
 
-            string contentMessage = message ?? $"Documents are not equal. Expected:{nl}{expectedDocument}{nl}but was:{nl}{actualDocument}";
-            string declarationMessage = message ?? $"Declarations are not equal. Expected: {expectedDocument.Declaration} but was {actualDocument.Declaration}";
+            string contentMessage = (message ?? string.Empty) + $"Documents are not equal. Expected:{nl}{expectedDocument}{nl}but was:{nl}{actualDocument}";
```

Comment:
Previously the assert messages would be non-actionable when specified a message.